src/components/react-admin/DraggableFormIterator.js

```diff
@@ -0,0 +1,282 @@
+/* eslint-disable
```

Comment:
Also, as in `/src/components/react-admin/CustomFormIterator.js`, please add a comment explaining in wich way the file diverge from original one.

Reply:
Added a comment to explain from what it has been implemented